src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf) format("ttf");
+  font-weight: 300;
+}
```

Comment:
check out point 7 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)